qiskit/transpiler/passmanager.py

```diff
@@ -121,8 +121,8 @@ def _finalize_layouts(self, dag):
         # Ordered list of original qubits
         original_qubits_reverse = {v: k for k, v in original_qubit_indices.items()}
         original_qubits = []
-        for i in range(len(original_qubits_reverse)):
-            original_qubits.append(original_qubits_reverse[i])
+        for v, k in original_qubits_reverse.items():
+            original_qubits.append(k)
```

Comment:
This looks like a behavioural change to me.  Dictionary iteration order is insertion order, but the previous form was specifically looking up in _numerical_ order of the `v` items.

The `v`s are ints here (from `0` to `n-1`) and the `k` are `Qubit` instances.

Reply:
ah, i see it now. ok, i'll have to update this